src/components/HomeSection.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  img: {
+    width: '100%',
+    height: '100%',
+    objectFit: 'contain',
+  },
+}));
+
+export default function ScholarshipsMadeSimpleContent(props) {
+  const classes = useStyles();
+  const { direction, tab, title, description, button, pic } = props;
+  return (
+    <Grid container spacing={2} direction={direction}>
```

Comment:
To vertically center the content
```suggestion
    <Grid container spacing={2} direction={direction} alignItem={center}>
```